Added zh i18n

DukeMay opened the pull request:
#6337 We need have portal in zh language
Please check it.I push the chinese translation.

gcusnieux commented:
Hi @DukeMay, thx for your contribution ! 
Could you push `zh` key in [environment files](https://github.com/gravitee-io/gravitee-portal-webui/tree/master/src/environments) ?

DukeMay commented:
Hi @gcusnieux , I will push zh key in environment.prod.ts or environment.ts?
Another, I have added a key to the local file.But it doesn't work.
gravitee-portal-webui/src/app/services/translation.service.ts in Line 32, I have changed it.
Please check it.
